src/sidebar/components/sidebar-content.js

```diff
@@ -325,6 +325,8 @@ function SidebarContentController(
     annotationUI.clearSelectedAnnotations();
     annotationUI.selectTab(selectedTab);
   };
+
+  this.viewSwitcherEnabled = features.flagEnabled('view-switcher');
```

Comment:
In theory feature flags can change after the `<sidebar-content>` is instantiated. The feature flags are stored in the app state, so a way to avoid this is to set `viewSwitcherEnabled` in the `annotationUI.subscribe` callback near the top of this function. You can use `isFeatureEnabled(state, featureName)` selector to extract the state of a feature flag from the app state.